src/experimental/constructs/error-budget-alarm.test.ts

```diff
@@ -16,6 +20,66 @@ describe("The ErrorBudgetAlarmExperimental construct", () => {
     expect(Template.fromStack(stack).toJSON()).toMatchSnapshot();
   });
 
+  it("should create the correct resources for a simple Availability SLO", () => {
+    const stack = simpleGuStackForTesting();
+    const ec2App = new GuEc2App(stack, {
+      applicationPort: 3000,
+      app: "test-gu-ec2-app",
+      access: { scope: AccessScope.PUBLIC },
+      instanceType: InstanceType.of(InstanceClass.T4G, InstanceSize.MEDIUM),
+      monitoringConfiguration: { noMonitoring: true },
+      userData: "#!/bin/dev foobarbaz",
+      certificateProps: {
+        domainName: "code-guardian.com",
+      },
+      scaling: {
+        minimumInstances: 1,
+      },
+    });
+    new GuErrorBudgetAlarmExperimental(stack, {
+      sloName: "MapiFrontsAvailability",
+      sloTarget: 0.999,
+      badEvents: new MathExpression({
+        expression: "loadBalancer5xxErrors + instance5xxErrors",
+        usingMetrics: {
+          loadBalancer5xxErrors: ec2App.loadBalancer.metricHttpCodeTarget(HttpCodeTarget.TARGET_5XX_COUNT),
+          instance5xxErrors: ec2App.loadBalancer.metricHttpCodeElb(HttpCodeElb.ELB_5XX_COUNT),
+        },
+      }),
+      validEvents: ec2App.loadBalancer.metricRequestCount(),
+      snsTopicNameForAlerts: "test-sns-topic",
+    });
+    expect(Template.fromStack(stack).toJSON()).toMatchSnapshot();
+  });
+
+  it("should create the correct resources for a simple Latency SLO", () => {
+    const stack = simpleGuStackForTesting();
+    const ec2App = new GuEc2App(stack, {
+      applicationPort: 3000,
+      app: "test-gu-ec2-app",
+      access: { scope: AccessScope.PUBLIC },
+      instanceType: InstanceType.of(InstanceClass.T4G, InstanceSize.MEDIUM),
+      monitoringConfiguration: { noMonitoring: true },
+      userData: "#!/bin/dev foobarbaz",
+      certificateProps: {
+        domainName: "code-guardian.com",
+      },
+      scaling: {
+        minimumInstances: 1,
+      },
+    });
+    new GuErrorBudgetAlarmExperimental(stack, {
+      sloName: "MapiFrontsLatency",
+      sloTarget: 0.999,
+      badEvents: ec2App.loadBalancer.metricTargetResponseTime({
+        statistic: "TC(0.5:)", // This gets a count of slow requests i.e. the number of requests that completed in 0.5 seconds or more
```

Comment:
N.B. it's possible to perform the entire calculation `(badEvents/validEvents) * 100` using the [PR statistic](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Statistics-definitions.html) `PR(0.5:)`.

Using `TC` allows us to use a standard construct regardless of the SLI type and also means that users can provide a more complex definition of valid events (e.g. using metric math to exclude certain response codes).